### ISSUE TITLE
Not compatible with dev tools

### ISSUE THREAD

**Issue by nikhil-dhiman** [alibaba/jetcache#85]
Hi, I am getting ClassCast Exception because of dev tools enabled. 
I can't stop using dev tools, since i am heavily dependant on this module during development phase.
Is there an option available in Jetcache to disable @Cached for profile.active = dev

**Comment by areyouok** [comment]
can you set *restart.enabled*=false? 
```
System.setProperty("spring.devtools.restart.enabled", "false");
```

**Comment by nikhil-dhiman** [comment]
Hi, I understand. but is there a way I can make @cached not working on the basis of spring.profile.active

Just like until @EnableMethodCache is on, @cached will not work.

**Comment by areyouok** [comment]
Thanks. I have fix it but need verification.
And I add ```jetcache.enableMethodCache``` to disable method cache in profile.

**Comment by Ikki-Dai** [comment]
Hi, I think this will useful to you

**Comment by areyouok** [comment]
fixed in 2.5.6
